hubmap_commons/string_helper.py

```diff
@@ -56,4 +59,74 @@ def listToCommaSeparated(lst, quoteChar = None, trimAndUpperCase = False):
 def allIndexes(value, character):
     return [i for i, ltr in enumerate(value) if ltr == character]
 
+def convert_str_literal(data_str):
+    if isinstance(data_str, str):
+        """
+        This was copied from:
+        https://github.com/hubmapconsortium/entity-api/blob/a832a906124623a889a943c15ff7c8d93f2bb068/src/schema/schema_manager.py#L1666
+        """
+        data_str = "".join(char for char in data_str if unicodedata.category(char)[0] != "C")
+        try:
+            data = ast.literal_eval(data_str)
+
+            if isinstance(data, (list, dict)):
+                return data
+        except (SyntaxError, ValueError, TypeError) as e:
+            # f"Invalid expression (string value): {data_str} to be evaluated by ast.literal_eval()"
+            pass
```

Comment:
@ChuckKollar @yuanzhou Should we be throwing an exception here to let the caller know that there is a problem?